### ISSUE TITLE
removeEventListener result

### ISSUE THREAD

**Issue by sinyakin** [java-native/jssc#63]
1. removeEventListener method has comment "If the operation is successfully completed, the method returns true, otherwise false"
But this method never return false, only true or exception.

2. Sometimes I get "Event listener already added" error, but I don't know why.
every time after working with port I perform removeEventListener. Can you help me?

**Comment by tresf** [comment]
>  the method returns true, otherwise false

Agreed that appears to be incorrect.  Instead it either removes the listener or throws an exception.  The documentation is incorrect.

> Sometimes I get "Event listener already added" error, but I don't know why.
every time after working with port I perform removeEventListener. Can you help me?

A unit test is required to know for sure, so you should provide a small code snippet which reproduces this.  Worth noting, closing the port implicitly removes the listener, so you should not attempt to remove a listener after closure.  The exception thrown in this scenario isn't the same as you're mentioning, so the best way to reproduce the issue is to know how to reproduce it.

https://github.com/java-native/jssc/blob/6d097525b4d02d620382842b4e72c4eadb838d44/src/java/jssc/SerialPort.java#L1088-L1092
